src/components/ArticleCard.jsx

```diff
@@ -1,7 +1,12 @@
+import { Link } from 'react-router-dom';
+import { Routes, Route} from 'react-router-dom'
+import { BrowserRouter } from 'react-router-dom'
+
 const ArticleCard = ({article}) => {
     return (
         <div>
-            <h2>Title: {article.title}</h2> 
+            <h2><Link to={`/articles/${article.article_id}`}>{article.title}</Link></h2>
+            <img src={article.article_img_url} />
             <h3>Topic: {article.topic}</h3>
             <h3>Author: {article.author}</h3>
```

Comment:
we should also show the when it was created in a more human readable format - we might want to tell the user how many votes/comments the article has as well